data/formats-data.js

```diff
@@ -3885,7 +3892,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	drifloon: {
-		randomBattleMoves: ["shadowball", "substitute", "calmmind", "hypnosis", "hiddenpowerfighting", "thunderbolt", "destinybond", "willowisp"],
+		randomBattleMoves: ["shadowball", "recycle", "calmmind", "acrobatics", "hex", "destinybond", "willowisp"],
```

Comment:
substitute is used for the sub recycle acrobatics physical set

Reply:
i dont trust good sets to come from its inclusion

Reply:
we just dont want the special set to have substitute lmao

Reply:
which will be pretty common and there isn't a good rule to make it work and not screw everything else